specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/examples/TableOperationPutOrPatchAcls.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res3376",
+    "accountName": "sto328",
+    "tableName": "table6185",
+    "api-version": "2020-08-01-preview",
+    "monitor": "true",
+    "parameters": {
+      "properties": {
+        "signedIdentifiers": [
+          {
+            "id": "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTI",
```

Comment:
where's the id come from？
How can user prepare the value of ID when send request?

Reply:
Same as what we have for File Service